lib/slug.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Helper class to generate slugs out of names or titles.
+class Slug
+  CONTROL_CHARACTERS_RE = /[\x00-\x1F\x7F]/
```

Comment:
Interested in what this does...

Reply:
It matches [the ASCII control characters](https://en.wikipedia.org/wiki/Control_character), things like the C string ending zero, the audible bell, etc.